### PR TITLE
feat: Update contributors to the packaging giudebook

### PR DESCRIPTION
**Pull request by lwasser**
To support a new release of the packaging guide, let's make sure that we have all new contributors represented in our zenodo file

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
       "name": "Nicholson, David",
       "orcid": "0000-0002-4261-4719"
     },
+    {
+      "affiliation": "",
+      "name": "Contributor, New",
+      "orcid": "orcid-here"
+    },
```
Comment (flpm) [inline .zenodo.json:17]:
```suggestion
    {
      "affiliation": "",
      "name": "Contributor, New",
      "orcid": "orcid-here"
    },
    {
      "affiliation": "",
      "name": "Moreno, Felipe"
    },
```

**Review by flpm — COMMENTED** [review]
I was not in the file, so I added an entry after the New Contributor one.

**Comment by jhkennedy** [comment]
[My reccord](https://github.com/pyOpenSci/python-package-guide/blob/e2e4caa21510b0583af39b1f74700cbb3edec006/.zenodo.json#L189-L193) could be updated to:
```diff
     {
-      "type": "Other",
-      "affiliation": "",
+      "affiliation": "Alaska Satellite Facility, University of Alaska Fairbanks",
       "name": "Kennedy, Joseph H.",
+      "orcid": "0000-0002-9348-693X"
     },
```
Or if that affiliation is too long, just "Alaska Satellite Facility".

**Comment by lwasser** [comment]
thank you @jhkennedy @flpm i don't think any additions can be too long as long as we follow the proper JSON formatting!

**Comment by lwasser** [comment]
ok i've added both new entries. @jhkennedy that's the first orcid i've seen with an "x" at the end or letters in general. i'm curious if that's something special or just the fact that they need more values for orcids?

**Comment by jhkennedy** [comment]
> @jhkennedy that's the first orcid i've seen with an "x" at the end or letters in general. i'm curious if that's something special or just the fact that they need more values for orcids?

Yep, it's weird! No idea why, but it is correct: https://orcid.org/0000-0002-9348-693X

**Comment by jhkennedy** [comment]
Okay I googled it:
> The last character of an ORCID identifier is a Checksum. It ranges in values from “0” - “10,” with X representing the value “10.” The Checksum ensures the other 15 digits of your identifier form a coherent iD.

https://support.orcid.org/hc/en-us/articles/360053289173-Why-does-my-ORCID-iD-have-an-X#:~:text=ORCID%20iD's%20that%20end%20with,identifier%20form%20a%20coherent%20iD.

**Comment by willingc** [comment]
Added a pre-commit.ci autofix label to fix the issue that prettier had. Once this is all green, I will merge. Thanks @lwasser.

**Pull request merged**